geode-apis-compatible-with-redis/README.md

```diff
@@ -1,29 +1,28 @@
-# Apache Geode APIs Compatible with Redis
+# <a name="top"></a>Apache Geode APIs Compatible with Redis
 
 Note: This feature is experimental and is subject to change in future releases of Apache Geode.
 
-[Introduction](#introduction)  
-[How To Try It](#how-to-try-it)  
-[Building Apache Geode](#building-apache-geode)  
-[Starting a Geode Server with Redis Enabled](#starting-a-server)  
-[Adding an Additional Geode Server Compatible with Redis](#adding-a-server)  
-[Shutting Down](#shutting-down)  
-[Redis Commands](#redis-commands)
+1. [Introduction](#introduction)  
+2. [How To Try It](#how-to-try-it)  
+    i. [Starting a Geode Server with APIs Compatible with Redis Enabled](#starting-a-server)  
+    ii. [Adding an Additional Geode Server Compatible with Redis](#adding-a-server)  
```

Comment:
Could we make the Geode Server part of these two consistent?  So make them both `Geode Server Compatible with Redis` or something

Reply:
"Geode Server Compatible with Redis" is certainly simpler than "Geode Server with APIs Compatible with Redis Enabled". I'm for it, if the simplification doesn't violate any trademark agreements.